ext/opentelemetry-ext-asgi/src/opentelemetry/ext/asgi/version.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__version__ = "0.4.dev0"
```

Comment:
start at `0.1.dev0`? I'm not super familiar with this project's convention however...

Reply:
Neither am I, so I decided to leave it for the review / feedback.

Reply:
But it does seem like all the `version.py` files are consistent on "0.4.dev0":
``
$ find ext/ -name "version.py" | xargs cat | grep "__version__ = "
__version__ = "0.4.dev0"
__version__ = "0.3.dev0"
__version__ = "0.4.dev0"
__version__ = "0.4.dev0"
__version__ = "0.4.dev0"
__version__ = "0.4.dev0"
__version__ = "0.4.dev0"
__version__ = "0.4.dev0"
__version__ = "0.4.dev0"
__version__ = "0.4.dev0"
__version__ = "0.4.dev0"
__version__ = "0.4.dev0"
```
The `0.3.dev0` tag is from: `ext-zipkin`.

Reply:
Perhaps a contributor could weigh in? If it's just unintentional copy and paste, seems better to address that sooner rather than later

Reply:
it is currently intentional to version all packages the same, as it correlates well with the source control tag and we publish all releases at the same time.

We could definitely amend this process, but the plan for now is to start splitting packages and version out once 1.0 is reached.